druntime/src/core/stdc/config.d

```diff
@@ -260,6 +260,23 @@ else version (DigitalMars)
         else version (Darwin)
             alias real c_long_double;
     }
+    else version (AArch64)
+    {
+        version (linux)
+            alias real c_long_double;
+        else version (FreeBSD)
+            alias real c_long_double;
+        else version (OpenBSD)
+            alias real c_long_double;
+        else version (NetBSD)
+            alias real c_long_double;
+        else version (DragonFlyBSD)
+            alias real c_long_double;
+        else version (Solaris)
+            alias real c_long_double;
+        else version (Darwin)
+            alias real c_long_double;
```

Comment:
It's actually `double` for all Apple and Microsoft targets.

Reply:
config.d sets c_long_double to real for all targets for GNU, LDC and SDC.

Anyhow, right now I'm working on a Linux target. I'll worry about the Mac and Windows targets when I'm working on them.